.github/workflows/publish_plugin_to_hub.yml

```diff
@@ -121,4 +121,4 @@ jobs:
         run: |
           mkdir -p ~/.local/share/cloudquery
           echo ${{ secrets.CQ_CI_CLOUDQUERY_HUB_TOKEN }} > ~/.local/share/cloudquery/token
-          cloudquery publish cloudquery/${{ needs.prepare.outputs.plugin_name }} --finalize
+          cloudquery plugin publish cloudquery/${{ needs.prepare.outputs.plugin_name }} --finalize
```

Comment:
For this to work we actually need to release a new CLI version and update it in the workflow here https://github.com/cloudquery/cloudquery/blob/8946c36362c4344115e015f0716b8aafb7cabd83/.github/workflows/publish_plugin_to_hub_duckdb.yml#L96

Reply:
See https://github.com/cloudquery/cloudquery/actions/runs/6562442141/job/17824393847#step:11:7.

I'll release https://github.com/cloudquery/cloudquery/pull/14659 and renovate should handle the update for us